EIPS/eip-2537.md

```diff
@@ -343,42 +343,17 @@ Required properties for pairing operation:
 - Degeneracy `e(P, 0*Q) = e(0*P, Q) = 1` 
 - Bilinearity `e(a*P, b*Q) = e(a*b*P, Q) = e(P, a*b*Q)` (internal test, not visible through ABI)
 
-Test vector for all operations are expanded in this `csv` files in [repo](https://github.com/matter-labs/eip1962/tree/master/src/test/test_vectors/eip2537).
+Test vector for all operations are expanded in this `csv` files in [repo](https://github.com/matter-labs/eip1962/tree/master/src/test/test_vectors/eip2537) as hex encodings of the inputs and outputs to the corresponding precompiles.
```

Comment:
This should either contain an inline set of test vectors, a link to a set of test vectors in in ../assets/eip-2537/* or be removed. External links should not be included in EIPs.